js/index.js

```diff
@@ -1 +1,59 @@
-// Your code goes here
+// Quan's code goes here
+const botBtn = document.querySelectorAll(".btn");
+for(let i=0; i<botBtn.length; i++){
+    botBtn[i].addEventListener('mousedown', (event) => {
+        event.target.style.color='#17A2B8';
+        event.target.style.backgroundColor= 'white';
+      });
+      botBtn[i].addEventListener('mouseup', (event) => {
+        event.target.style.color='white';
+        event.target.style.backgroundColor= '#17A2B8';
```

Comment:
Nice job using a for loop here to access all of the bottom buttons at once and I like the effect you added here! 👍